Flagsmith.Client.Test/HttpMocker.cs

```diff
@@ -1,18 +1,27 @@
-﻿using Moq;
-using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿using System.Collections.Generic;
+using System.Net;
 using System.Net.Http;
-using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using System.Web;
-using System.Net;
+using Moq;
 
 namespace Flagsmith.FlagsmithClientTest
 {
     internal static class HttpMocker
     {
+        public static Mock<HttpClient> MockHttpResponse(HttpStatusCode statusCode, string content)
+        {
+            var httpClientMock = new Mock<HttpClient>();
+            httpClientMock.Setup(x => x.SendAsync(It.IsAny<HttpRequestMessage>(), It.IsAny<CancellationToken>()))
+                .Returns(() => Task.FromResult(new HttpResponseMessage()
```

Comment:
This is necessary since the `MockHttpResponse` used by other methods is not thread-safe (since `StringContent` is not thread-safe).

This method thus does not keep returning _the same response_ but a new response / set of content on each invocation.